docs/firmware/device-sdk/logging/concepts-logging.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Concepts - Logging

```

Comment:
These are fine to leave for now but would like them to drop the "Concepts - " and be more descriptive, like "Sending logs."